wd_helper.au3

```diff
@@ -1082,8 +1082,8 @@ Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
 
 	Local $iExt = $_WD_HTTPRESULT
 
-	; if $iSelectedOption was found then pass the ID ($iSelectedOption) to @extended
-	If $_WD_HTTPRESULT = $HTTP_STATUS_OK And $iSelectedOption <> -1 Then $iExt = $iSelectedOption
+	; if OPTIONS2 was succesfully used then pass the ID ($iSelectedOption) to @extended
```

Comment:
Description in Function Header should be added.
@Danp2  can I leave this part of the job to you ?

Reply:
This is no longer needed.